js/main.js

```diff
@@ -0,0 +1,26 @@
+var xhrFilms = new XMLHttpRequest();
+xhrFilms.open('GET', 'https://ghibliapi.herokuapp.com/films');
+xhrFilms.responseType = 'json';
+xhrFilms.addEventListener('load', function () {
+
+  for (var i = 0; i < xhrFilms.response.length; i++) {
+    var titleElement = document.createElement('div');
+    titleElement.setAttribute('class', 'titles');
+    var titles = xhrFilms.response[i].title;
+    titleElement.textContent = titles;
+    var imageName = titles.toLowerCase().replace(/\s/g, '');
+    var titleImage = document.createElement('img');
+    titleImage.setAttribute('src', 'images/' + imageName + '.jpeg');
+    var $animationTitle = document.querySelector('.animationTitle');
+    var titleImages = document.createElement('div');
+    titleImages.setAttribute('class', 'imageLocation');
+    titleImage.setAttribute('width', '100px');
+    titleImage.setAttribute('height', '100px');
+    titleImage.setAttribute('class', 'img');
```

Comment:
Why not use a class to set the height and width?

Reply:
updated with using class! thanks